Board.cs

```diff
@@ -1,18 +1,64 @@
+using System.Collections.Generic;
+using System.Linq;
+
 namespace TicTacToe
 {
     public class Board
     {
         private static IConsoleWriter _writer;
-        static char[] boardArr = { '1', '2', '3', '4', '5', '6', '7', '8', '9' };
+        private static List<object> _boardList;
 
         public Board(IConsoleWriter IWriter)
         {
+            _boardList = new List<object>(){ 1, 2, 3, 4, 5, 6, 7, 8, 9 };
             _writer = IWriter;
         }
         
         public void DisplayBoard()
         {
-            _writer.PrintBoard(boardArr);
+            _writer.PrintBoard(_boardList);
+        }
+
+        public void MakeMove(int cell, char playerSymbol)
+        {
+            _boardList[cell - 1] = playerSymbol;
+        }
+
+        public bool IsValidMove(int cell)
+        {
+            if (Enumerable.Range(1, 9).Contains(cell))

```

Comment:
This is also very coupled to a 3x3 implementation. How could you make it more flexible?